x-pack/plugins/security_solution/public/cases/components/bulk_actions/index.tsx

```diff
@@ -26,34 +27,72 @@ export const getBulkItems = ({
   selectedCaseIds,
   updateCaseStatus,
 }: GetBulkItems) => {
+  let statusMenuItems: JSX.Element[] = [];
+
+  const openMenuItem = (
+    <EuiContextMenuItem
+      data-test-subj="cases-bulk-open-button"
+      disabled={selectedCaseIds.length === 0}
+      key="cases-bulk-open-button"
+      icon={statuses[CaseStatuses.open].icon}
+      onClick={() => {
```

Comment:
please put into `useCallback`

Reply:
I get this error. I think is ok to leave it as it is as I have to create three useCallbacks, one for each status change.

```
React Hook "useCallback" is called in function "getBulkItems" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter.
```